exchanges/interfaces.go

```diff
@@ -87,4 +86,8 @@ type IBotExchange interface {
 	// pair,asset, url/proxy or subscription change
 	FlushWebsocketChannels() error
 	AuthenticateWebsocket() error
+	// Exchange order related execution limits
+	GetLimit(a asset.Item, cp currency.Pair) (*order.Limits, error)
```

Comment:
I think GetLimit is too ambiguous as it could mean many different things. Would be nice to reference order execution limit in its name. `GetOrderExecutionLimits`.